lib/palindrome_permutation.rb

```diff
@@ -1,4 +1,24 @@
 
 def palindrome_permutation?(string)
```

Comment:
This is another solution that's similarly inefficient.  Can you think of a hash-based solution that's more efficient?

Reply:
I believe I fixed this. What is the proper way to send it back to you, if needed?